ComputerVisionServer/moedx/client/remote_bench.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import os
+import json
+import time
+import logging
+import argparse
+import requests
+from threading import Thread
+
+LAUNCH_INTERVAL = 1 # Seconds
+# filename = "objects_320x180_x2.mp4"
+filename = "objects_320x180.mp4"
+# filename = "objects.mp4"
+
+def launch_remote_benchmark(url):
+    body = '-s 80.187.140.9 -e /object/detect/ -c websocket -f %s -n PING --server-stats' %filename
+    # body = '-s cv-gpu-cluster.frankfurt-main.tdg.mobiledgex.net --tls -e /object/detect/ -c websocket -f %s -n PING --server-stats' %filename
+    # body = '-s cv-gpu-cluster.frankfurt-main.tdg.mobiledgex.net --tls -e /object/detect/ -c websocket -f %s -n PING --fullsize' %filename
+    # body = '-s cv-gpu-cluster.frankfurt-main.tdg.mobiledgex.net --tls -e /object/detect/ -c websocket -f %s -n PING' %filename
+    # body = '-s cv-gpu-cluster.frankfurt-main.tdg.mobiledgex.net --tls -e /object/detect/ -c rest -f %s -n PING --skip-frames 4' %filename
+    response = requests.post("%s/client/benchmark/" %url, data=body)
+    print("Response for %s:" %url)
+    print(response.content.decode("utf-8"))
+
+def remote_download(url):
+    # params = {'url': 'http://acrotopia.com/mobiledgex/%s' %filename}
+    params = {'url': 'http://opencv.facetraining.mobiledgex.net/videos/landscape/%s' %filename}
+    response = requests.post("%s/client/download/" %url, data=params)
+    print("Response for %s:" %url)
+    print(response.content.decode("utf-8"))
+
+urls = [
```

Comment:
are you going to manually maintain this list of cv instances? Maybe we could do a GetAppInstList once in a while?

Reply:
Yes, this is manual. It's set up so I can easily perform a 5-client test, a 4-client test, etc., by commenting or uncommenting code.

Passing in the number of desired clients and doing a GetAppInstList would be a nice improvement, but this benchmarking code might never get used again after I collect my final numbers.

Reply:
Maybe say test only somewhere if leaving in?